numpy/core/shape_base.py

```diff
@@ -184,6 +184,10 @@ def vstack(tup):
     Take a sequence of arrays and stack them vertically to make a single
     array. Rebuild arrays divided by `vsplit`.
 
+    This function continues to be supported for backward compatibility, but
+    you should prefer ``np.concatenate`` or ``np.stack`` (keep in mind that
+    ``np.stack`` was added in numpy version 1.10).
```

Comment:
@charris I like your first suggestion:

    This function continues to be supported for backward compatibility, but
    you should prefer ``np.concatenate`` or ``np.stack``. The ``np.stack``
    function was added in NumPy 1.10.